### PR TITLE
Fixes support form

### PR DESCRIPTION
**Pull request by thiessenp-cds**
# Summary | Résumé

Fixes support form to allow a submission.

## Test

1. Go to the support form (link in the page footer)
2. Fill out the form but try to make it obvious this is a test
3. Submit the form
4. Success screen should show
5. Check the support slack channel and flag this as a test

Next do the same but for the contact us form. You can get there by going to the support form and clicking on the link for the contact us page.

### CODE REVIEW

**Comment by craigzour** [comment]
Did not get a success screen :(

Screenshot 2024-02-01 at 10 19 02 AM

File: app/(gcforms)/[locale]/(form administration)/form-builder/support/[[...supportType]]/page.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default async function Page({
 
   return (
     <DefaultLayout showLanguageToggle>
-      <ClientSide supportType={supportType ? supportType[0] : ""} />
+      <ClientSide supportType={supportType ? supportType[0] : "support"} />
```
Comment (wmoussa-gc) [inline app/(gcforms)/[locale]/(form administration)/form-builder/support/[[...supportType]]/page.tsx:32]: so we introducing a default?

**Comment by thiessenp-cds** [comment]
> Did not get a success screen :(
> 
> Screenshot 2024-02-01 at 10 19 02 AM

Wops, looks like the Freshdesk API key needs to be updated

**Comment by thiessenp-cds** [comment]
> Did not get a success screen :(
> 
> Screenshot 2024-02-01 at 10 19 02 AM

Good catch :) I put a breakpoint to stop execution locally before sending the ticket. Assumed that part would work (no change). But it seems like the `FRESHDESK_API_KEY` is undefined - looking into it now

File: app/(gcforms)/[locale]/(form administration)/form-builder/support/[[...supportType]]/page.tsx (hunk 1 above)
Comment (thiessenp-cds) [reply app/(gcforms)/[locale]/(form administration)/form-builder/support/[[...supportType]]/page.tsx:32]: Sort of : ) I suppose the default is undefined in which case a not found is shown.
This code originally made more sense to be combined and handle both support and contact us "pages. It's changed over time though and IMO has become difficult to maintain. I'd like to refactor this out into a separate support and a separate contact us page in the future.

**Comment by thiessenp-cds** [comment]
Looks like an issue with the environment - works locally. I'll close this for now

**Pull request closed without merging**